model/connection_string.go

```diff
@@ -162,7 +163,11 @@ func (cs ConnectionString) String() string {
 		case []byte:
 			res = append(res, tag+"="+base64.StdEncoding.EncodeToString(typ))
 		case crypto.String:
-			res = append(res, tag+"="+base64.StdEncoding.EncodeToString([]byte(typ)))
+			value := base64.StdEncoding.EncodeToString([]byte(typ))
```

Comment:
Technically, you only need to encode the first three characters (resulting in 4 base64 chars).

Reply:
I force-pushed doing what you are suggesting, thanks.